### PR TITLE
Mobile Portfolio First Page 

### PR DESCRIPTION
**Pull request by AbhishekkM05**
Made First Page of my Mobile portfolio project with HTML and CSS.
Added Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+        <header>
+        <nav>
+        <div class="Main-container"><p>My Logo</p>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required for the logo on the page to have a `link` tag. Kindly  surround your text that serves as logo with `<a>` tag. Something like this. `<p><a>My Logo</a></p>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="menu"><i class="fa fa-bars"></i></div>
```
Comment (Qoosim) [inline index.html]: - [x] It is required for your hamburger to be inside a `nav` tag. Kindly change 
`<div class="menu"><i class="fa fa-bars"></i></div>` 
  to
 `<nav class="menu"><i class="fa fa-bars"></i></nav>`

**Review by Qoosim — CHANGES_REQUESTED** [review]
#   *Status: Required Changes* :recycle: 

Hi @AbhishekkM05, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GiHub Flow. :heavy_check_mark: 
-   Meaningful commict messages. :heavy_check_mark: 
-   Fair README file. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Kindly produce a close replica of your chosen figma template. This involves, font-size, font-weight, color, margin and padding among other. You've put in a lot of effort to push your portfolio this far, but I believe you can do better. Kindly look at below screenshots of both your design at this moment and the figma template you chose and ensure you produce a close replica of your chosen template for your project to be due for the approval.
- [x] Look and compare your header and headline with your chosen template. :point_down: 

<table>
   <tr>
      <th>Figma Template</th>
      <th>Your Template</th>
   </tr>
   <tr>
      <td>
          
</td>
      <td>
          
</td>
   </tr>
</table>


- [ ] Kindly look at  your let connect and social icons and compare them with the ones in your chosen template.

<table>
   <tr>
      <th>Figma Template</th>
      <th>Your Template</th>
   </tr>
   <tr>
      <td>
          
</td>
      <td>
          
</td>
   </tr>
</table>


- [ ] Kindly look at the overall outlook of your design and that of the template you chose and compare them in terms of font-size, font-weight, color, space, line-height, margin, and padding among other things and ensure you produce a close replica of the design. :pray: 

<table>
   <tr>
      <th>Figma Template</th>
      <th>Your Template</th>
   </tr>
   <tr>
      <td>
          
</td>
      <td>
          
</td>
   </tr>
</table>

- [ ] Kindly customize your README file to contain relative information about your project and how it can be used by others.

File: index.html (hunk 1 above)
Comment (AbhishekkM05) [reply index.html]: how to create that time and battery symbols on top of my page ??
also due to image i am not able to align texts similar to figma design .

File: index.html (hunk 2 above)
Comment (AbhishekkM05) [reply index.html]: it is already inside the nav

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+        <header>
+        <nav>
+        <nav class="Main-container"><p class="logo"><a="">My Logo</a></p>
```
Comment (Nemwel-Boniface) [inline index.html:18]: - [x] [OPTIONAL] To completely make your logo a link, add the href to it. i.e ```        <nav class="Main-container"><p class="logo"><a href="#">My Logo</a></p>```

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @AbhishekkM05         :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (AbhishekkM05) [reply index.html:18]: thanks. i was not putting # and getting a underline in My logo

Comment (Nemwel-Boniface) [reply index.html:18]: @AbhishekkM05 the underline can be removed by CSS with the ```text-decoration: none;```. Hope this helps!

**Pull request merged**